### ISSUE TITLE
[A11Y] Heading level is missing for “Get our newsletter” section heading present on Blog post template

### ISSUE THREAD

**Issue by gkhadke** [ampproject/ampstart#379]
**ENVIRONMENT:**

OS (number): N (7.1.2)
Build number: N2G47H
Form factor (Smartphone / tablet):  Nexus 6P
Browser: Chrome Dev 59.0.3056.4

**Pre-requisite:**
1. Talkback is ON

**STEPS TO REPRODUCE:**

1. Open [https://ampstart.com](https://ampstart.com)
2. Linearly navigate to the “Blog Post” template and double tap on View Template
3. Change the reading granularity of talkback to “Headings and Landmark” by swiping “top to bottom” by using one finger and try to navigate through the headings/landmarks on the page.

**ACTUAL RESULTS:**

Talkback skips the “Get our newsletter” section heading as heading level is not assigned to it.

**EXPECTED RESULTS:**

Appropriate heading level should be assigned, so that talkback and other screen readers doesn’t skip the heading.

**ATTACHMENT(S):**

<img width="1440" alt="getournewsletter_missingheading" src="https://cloud.githubusercontent.com/assets/26986019/24820134/616efba2-1b9c-11e7-85f1-4b5440aa84df.png">

**Comment by spotukuchi** [comment]
This issue is Reproducible in the below environment:

ENVIRONMENT:
Form factor : iPAD
Model : MC705LL/A
Version : 9.3.5(13G36)
Browser : Safari

Pre-requisite:
Turn 'VoiceOver' ON. ( Settings->General->Accessibility->VoiceOver->ON )

STEPS TO REPRODUCE:
1. Open https://ampstart.com
2. Linearly navigate to the “Blog Post” template and double tap on View Template
3. Change the reading granularity to “Headings" and try to navigate through the headings on the page.

ACTUAL RESULTS:
Voice Over skips reading "Get our Newsletter" as it is not assigned a heading level.

EXPECTED RESULTS:
Appropriate heading level should be assigned, so that VoiceOver and other screen readers doesn’t skip the heading.

ATTACHMENTS:
Please see the video in the link below for reference:

https://drive.google.com/corp/drive/folders/0BwgKOOZP99EuUzVWT25TVzl2S00
